### PR TITLE
docs(LinuxCommand): update nl, pwd and whereis

### PR DESCRIPTION
**Pull request by Chengtian0730**
Description
----
update linux command nl, pwd and whereis.

Checklist
----
- [x] update functions, notes and examples of nl
- [x] update functions and examples of pwd
- [x] update functions and examples of whereis

### CODE REVIEW

File: docs/LinuxCommand/whereis.md [hunk 1]
```diff
@@ -29,3 +31,15 @@ whereis命令只能用于程序名的搜索，而且只搜索二进制文件（
 和find相比，whereis查找的速度非常快，这是因为linux系统会将系统内的所有文件都记录在一个数据库文件中，当使用whereis和下面即将介绍的locate时，会从数据库中查找数据，而不是像find命令那样，通过遍历硬盘来查找，效率自然会很高。但是该数据库文件并不是实时更新，默认情况下时一星期更新一次，因此，我们在用whereis和locate查找文件时，有时会找到已经被删除的数据，或者刚刚建立文件，却无法查找到，原因就是因为数据库文件没有被更新。
```
Comment (BobAnkh) [inline docs/LinuxCommand/whereis.md]: 不知道你有没有仔细阅读过这个内容，应该需要进行一些更改和重写，以更加适应当前仓库的文档组织形式，有些写法是因为这最初是我笔记的草稿才这么写的。需进行修改。

**Review by BobAnkh — CHANGES_REQUESTED** [review]

**Comment by BobAnkh** [comment]
Do you use the pull request template to organize this pull request?

See [Contributing](https://github.com/BobAnkh/LinuxBeginner/blob/master/CONTRIBUTING.md) and [README](https://github.com/BobAnkh/LinuxBeginner/blob/master/README.md#%E5%A6%82%E4%BD%95%E8%B4%A1%E7%8C%AE)

Also, you can find PULL_REQUEST_TEMPLATE [here](https://github.com/BobAnkh/LinuxBeginner/blob/master/.github/PULL_REQUEST_TEMPLATE.md). View [raw](https://raw.githubusercontent.com/BobAnkh/LinuxBeginner/master/.github/PULL_REQUEST_TEMPLATE.md) to see the comments in this template

**Comment by Chengtian0730** [comment]
I have modified whereis.md and this pull request. Please review. @BobAnkh

**Review by BobAnkh — APPROVED** [review]
A good job!

**Pull request merged**